cmd/kpod/build.go

```diff
@@ -0,0 +1,139 @@
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "refrain from announcing build instructions and image read/write progress",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Usage: "`path` to an alternate runtime",
+		},
+		cli.StringSliceFlag{
+			Name:  "runtime-flag",
+			Usage: "add global flags for the container runtime",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+		cli.StringSliceFlag{
+			Name:  "tag, t",
+			Usage: "`tag` to apply to the built image",
+		},
+		cli.BoolFlag{
+			Name:  "tls-verify",
+			Usage: "Require HTTPS and verify certificates when accessing the registry",
+		},
+	}
+	buildDescription = "This command launches the Buildah command to build an OCI Image. Buildah must be installed for this command to work."
+	buildCommand     = cli.Command{
+		Name:        "build",
+		Aliases:     []string{"build"},
+		Usage:       "Build an image using instructions in a Dockerfile",
+		Description: buildDescription,
+		Flags:       buildFlags,
+		Action:      buildCmd,
+		ArgsUsage:   "CONTEXT-DIRECTORY | URL",
+	}
+)
+
+func buildCmd(c *cli.Context) error {
+
+	budCmdArgs := []string{"bud"}
+
+	if c.IsSet("build-arg") {
+		budCmdArgs = append(budCmdArgs, "--build-arg")
+		budCmdArgs = append(budCmdArgs, c.StringSlice("build-arg")...)
+	}
+	if c.IsSet("file") || c.IsSet("f") {
+		for _, fileName := range c.StringSlice("file") {
+			budCmdArgs = append(budCmdArgs, "--file")
+			budCmdArgs = append(budCmdArgs, fileName)
+		}
+	}
+	if c.IsSet("format") {
+		budCmdArgs = append(budCmdArgs, "--format")
+		budCmdArgs = append(budCmdArgs, c.String("format"))
+	}
+	if c.IsSet("pull-always") {
+		budCmdArgs = append(budCmdArgs, "--pull-always")
+	}
+	if c.IsSet("quiet") {
+		budCmdArgs = append(budCmdArgs, "--quiet")
+	}
+	if c.IsSet("runtime") {
+		budCmdArgs = append(budCmdArgs, "--runtime")
+		budCmdArgs = append(budCmdArgs, c.String("runtime"))
+	}
+	if c.IsSet("runtime-flag") {
+		budCmdArgs = append(budCmdArgs, "--runtime-flag")
+		budCmdArgs = append(budCmdArgs, c.StringSlice("runtime-flag")...)
+	}
+	if c.IsSet("signature-policy") {
+		budCmdArgs = append(budCmdArgs, "--signature-policy")
+		budCmdArgs = append(budCmdArgs, c.String("signature-policy"))
+	}
+	if c.IsSet("tag") || c.IsSet("t") {
+		for _, tagName := range c.StringSlice("tag") {
+			budCmdArgs = append(budCmdArgs, "--tag")
+			budCmdArgs = append(budCmdArgs, tagName)
+		}
+	}
+	if c.IsSet("tls-verify") {
+		budCmdArgs = append(budCmdArgs, "--tls-verify")
+		if c.Bool("tls-verify") {
+			budCmdArgs = append(budCmdArgs, "True")
+		} else {
+			budCmdArgs = append(budCmdArgs, "False")
+		}
+	}
+
+	if len(c.Args()) > 0 {
+		budCmdArgs = append(budCmdArgs, c.Args()...)
+	}
+
+	buildah := "buildah"
+
+	_, err := exec.Command(buildah).Output()
+	if err != nil {
+		return errors.Wrapf(err, "buildah is not installed on this server")
```

Comment:
I think you should check for the existance of buildah to make this claim.  Buildah could throw a multiple errors.

Reply:
kk, I've added check for exec.LookPath for buildah, then kept this one but changed the wording slightly in the error.